backend/terraform/environments/example/main.tf

```diff
@@ -164,6 +166,12 @@ module "analyzer" {
 
   # List of service integration names. These are referenced in the services.json config.
   service-integrations = []
+
+  # Enable Forwarding for detected secrets
+  secrets_enabled = false
```

Comment:
I would set default values for secrets_enabled and ghas_enabled in the module. I think It makes sense for the other ones to be required though.